include/blast/math/simd/arch/Avx2.hpp

```diff
@@ -20,6 +20,15 @@
 
 namespace blast
 {
+    namespace detail
+    {
+        std::size_t constexpr registerCapacity(xsimd::avx2)
+        {
+            return 16;
```

Comment:
In bytes? Couldn't find the documentation on this function

Reply:
Ah, no, it's `num_of_vector_registers`. Better name?

Reply:
Agree

Reply:
Could be